vue/components/ui/molecules/animated-button-icon/animated-button-icon.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="animated-button-icon">
+        <button-icon
+            class="button button-icon-front"
+            v-bind:icon="icon"
+            v-bind:color="color"
+            v-bind:size="size"
+            v-bind:disabled="disabled"
+            v-bind:class="{ visible: !clicked }"
+            v-on:click="onClick"
+        />
+        <button-icon
+            class="button button-icon-click"

```

Comment:
button-icon-back? (as opposed to front)

I understand that for you click = animation, but I don't think that's clear for everyone reading this.